src/rp_turn/templates/nginx

```diff
@@ -39,37 +39,37 @@ server {
     error_page 404 /404.html;
     error_page 500 502 503 504 /50x.html;
 
-    # Redirect from web root to /webapp
-    location / {
-        return 301 /webapp$is_args$args;
-    }
-
     proxy_ssl_server_name on;
 
-{% for location in ["/api", "/webapp", "/static", "/teams"] %}
-{% set logfile = location.split("/")[-1] %}
-    location {{location}} {
+{% for location in ["", "api", "webapp", "webapp2", "static", "teams"] %}
```

Comment:
We should remove "webapp" and "webapp2" here as these location paths depend on what paths are configured for the deployment's webapps.
Sysadmins can add extra paths here if they wish to segregate logs for their different webapp paths.